samples/boards/mimxrt1170_evk_cm7/magic_addr/README.rst

```diff
@@ -0,0 +1,14 @@
+.. _flexram_magic_addr:
+
+FLEXRAM Magic Addr
+###########
+
+Overview
+********
+
+A sample that shows how to use RT11XX FLEXRAM Magic Addr functionality
```

Comment:
Please add more documentation here outlining what the magic address in the FlexRAM is, and what it does